python/humbug/report.py

```diff
@@ -74,6 +92,23 @@ def __init__(
 
         self.is_excepthook_set = False
 
+    def create_report(
+        self,
+        token: Union[str, uuid.UUID],
+        title: str,
+        content: str,
+        tags: List[str] = [],
+        **kwargs,
+    ):
+        method = "POST"
+        report_path = f"humbug/reports"
```

Comment:
Don't use `f`-strings. This will break in Python 3.6. Use `"<string>".format` instead.

Reply:
done